src/modules/commander/HealthAndArmingChecks/checks/modeCheck.cpp

```diff
@@ -77,29 +77,35 @@ void ModeChecks::checkAndReport(const Context &context, Report &reporter)
 
 	if (local_position_modes != NavModes::None) {
 		/* EVENT
+		 * @description
+		 * The available positioning data is not sufficient to execute the selected mode.
 		 */
 		reporter.armingCheckFailure(local_position_modes, health_component_t::local_position_estimate,
 					    events::ID("check_modes_local_pos"),
-					    events::Log::Error, "No valid local position estimate");
+					    events::Log::Error, "Navigation error: No valid local position estimate");
```

Comment:
Could we simplify by saying local position estimate = (any) position estimate, since we never have a local estimate without a global one? 🤔

Would bring me down to "Positioning insufficient, selected mode can't navigate."
This can be adapted for "global positioning" and "altitude estimate".

Reply:
> Could we simplify by saying local position estimate = (any) position estimate, since we never have a global estimate without a local one?

yes, then why not `Navigation error: No valid position estimate`

Reply:
Good idea, removed the "local".